docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -9,8 +9,7 @@ times, select the *Rule Monitoring* tab on the *Rules* page (*Detect* ->
 [role="screenshot"]
 image::images/monitor-table.png[]
 
-For detailed information on a rule, its generated alerts, and its errors, click on
-a rule name in the *All rules* table.
+For detailed information on a rule, its generated alerts, and its errors, click on a rule name in the *All rules* table. You can also perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```

Comment:
```suggestion
For detailed information on a rule, alerts it generated, and associated errors, click on its name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```
Nits. Mostly I just thought that "its generated alerts" was an awkward turn of phrase.

Reply:
Agree, with a couple slight amendments:
```suggestion
For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```